code/modules/robotics/robot/module_tool_creator/modules.dm

```diff
@@ -13,15 +13,16 @@
 		/obj/item/noisemaker,
 		/obj/item/robot_foodsynthesizer,
 		/obj/item/reagent_containers/food/drinks/bottle/beer/borg,
-		/obj/item/reagent_containers/food/drinks/drinkingglass,
+		/obj/item/reagent_containers/food/drinks/drinkingglass/pitcher,
```

Comment:
Can you confirm proper interaction with the soda/alcohol dispenser?

Reply:
Yes, the pitcher does work properly.